storage/client_test.go

```diff
@@ -267,6 +271,26 @@ func (m *multiTestContext) replicateRange(raftID int64, sourceStoreIndex int, de
 	})
 }
 
+func (m *multiTestContext) unreplicateRange(raftID int64, source, dest int) {
+	rng, err := m.stores[source].GetRange(raftID)
+	if err != nil {
+		m.t.Fatal(err)
+	}
+
+	err = rng.ChangeReplicas(proto.REMOVE_REPLICA,
+		proto.Replica{
+			NodeID:  m.idents[dest].NodeID,
+			StoreID: m.idents[dest].StoreID,
+		})
+	if err != nil {
+		m.t.Fatal(err)
+	}
+
+	// Removing a range doesn't have any immediately-visible side
+	// effects, (and the removed node may be stopped) so return as soon
+	// as the removal has committed on the leader.
+}
```

Comment:
As the comment says, the node may be stopped when it is removed (and it is, in this test). So even if we hooked this up it we couldn't wait for it to happen here. 

We could check for the removed data later, where I have the TODO comment in the other file. But it wouldn't be safe to do this yet; it would break important raft invariants if a node were removed from the group, deleted its data, then was re-added. Fixing this is my next big task.